File: BAEKJOON_10430_나머지.py
#(A+B)%C
def cal_1(A, B, C):
    return (A+B)%C

#((A%C)+(B%C))%C
def cal_2(A, B, C):
    return ((A%C)+(B%C))%C

#(A*B)%C
def cal_3(A, B, C):
    return (A*B)%C

#((A%C)*(B%C))%C
def cal_4(A, B, C):
    return ((A%C)*(B%C))%C

if __name__ == "__main__":
    A,B,C = map(int, input().split(' '))

    print(cal_1(A,B,C))
    print(cal_2(A,B,C))
    print(cal_3(A,B,C))
    print(cal_4(A,B,C))

File: 수학/BAEKJOON_10430_나머지.cpp
#include<iostream>
using namespace std;

//(A+B)%C
int cal_1(int A, int B, int C)
{
	return (A + B) % C;
}

//((A%C)+(B%C)%C)
int cal_2(int A, int B, int C)
{
	return ((A % C) + (B % C) % C);
}

//(A*B)%C
int cal_3(int A, int B, int C)
{
	return (A * B) % C;
}

//((A%C)*(B%C))%C
int cal_4(int A, int B, int C)
{
	return ((A % C) * (B % C)) % C;
}

int main(void)
{
	int A, B, C;

	cin >> A >> B >> C;

	cout << cal_1(A, B, C) << endl;
	cout << cal_2(A, B, C) << endl;
	cout << cal_3(A, B, C) << endl;
	cout << cal_4(A, B, C) << endl;

	return 0;
}